### ISSUE TITLE
Podcast duplicated

### ISSUE THREAD

**Issue by Chralu** [amugofjava/anytime_podcast_player#135]
**Describe the bug**
Sometimes, one of the subscribed podcast gets duplicated.

It appears then twice in the main screen. The episode list of each of the duplicated podcasts is not synchronized.

**To Reproduce**
I haven't found a sure way of reproducing that issue yet.

I try to get more info about this.

**Smartphone (please complete the following information):**
 - iPhone 8
 - Fairphone 3 (Android 13)

**Comment by amugofjava** [comment]
Hi @Chralu,

This bug has been fixed and will be included in the next release.

The duplication happened when Anytime tried to fetch a feed and it failed. As we force https, when it failed we downgraded to http and tried again. The problem was, Anytime was then storing this downgraded URL rather than the original. When you next fetched and update, it would try the https version again, succeed but see it as a different podcast and duplicate it.
